src/main/java/org/commonwl/view/git/GitDetails.java

```diff
@@ -242,4 +252,44 @@ public boolean equals(Object o) {
     public int hashCode() {
         return Objects.hash(repoUrl, branch, path, packedId);
     }
+
+    /**
+     * Retrieves license details from the repo, if present.
+     * @param workTree the path to the locally cloned repo
+     * @return The license URI
+     */
+    public String getLicense(Path workTree) {
+        try {
+            String[] command = {"licensee", "detect", "--json", workTree.toString()};
+            if (logger.isDebugEnabled()) {
+                logger.debug("Calling " + String.join(" ", command));
```

Comment:
Missing test coverage; also, perhaps better for the `trace` level?

Reply:
I put it as trace. The line is not covered because the log level is set to info now